### PR TITLE
Add tests to check the sanity for the database input methods

### PR DESCRIPTION
**Pull request by samyak-jn**
Hi :hugs:

This pull request is in with regards to story GITOPS-1375, which checks for inputting malicious values within the database objects!

### CODE REVIEW

File: backend-shared/config/db/injection_test.go [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+		EngineCluster_id:        gitopsEngineCluster.Gitopsenginecluster_id,
+	}
+
+	clusterAccess := ClusterAccess{
+		Clusteraccess_user_id:                   testClusterUser.Clusteruser_id,
+		Clusteraccess_managed_environment_id:    managedEnvironment.Managedenvironment_id,
+		Clusteraccess_gitops_engine_instance_id: gitopsEngineInstance.Gitopsengineinstance_id,
+	}
+
+	if err = dbq.CreateClusterCredentials(ctx, &clusterCredentials); err != nil {
```
Comment (jgwest) [inline backend-shared/config/db/injection_test.go:219]: These should all use this:
```
	if !assert.NoError(t, err) {
			return
		}
```
or at least `assert.NoError(t, err)`), even if this is already tested elsewhere.

**Review by jgwest — CHANGES_REQUESTED** [review]
Looks good, minor tweak then good to go.

**Review by samyak-jn — COMMENTED** [review]
Made the relevant changes (use of assert) where it was left, thanks!

**Review by jgwest — APPROVED** [review]
:sunglasses:

**Pull request merged**